source/package.json

```diff
@@ -18,7 +18,7 @@
     "cdk": "cdk ",
     "diff": "cdk diff QCStack",
     "synth": "cdk synth QCStack ",
-    "deploy": "cdk deploy QCStack",
+    "deploy": "./cdk_deploy.sh",
```

Comment:
why we need this? I don't think it's necessary

Reply:
removed